### PR TITLE
[Task]: Add French translations to aria-labels

### PR DESCRIPTION
**Pull request by KobikahChandran**
### What this PR accomplishes

- Provides French translations to aria-labels on French pages

### Issue(s) addressed
DATA-1365

### What needs review

- French translations correspond to the correct English labels and spelled correctly (accents)
- Translations available on necessary pages
- There are several "Clear field" labels, please check if they all show the French translation

### CODE REVIEW

File: ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot [hunk 1]
```diff
@@ -1933,3 +1933,35 @@ msgstr ""
...
+#: ckanext/ontario_theme/templates/internal/header.html:152
+msgid "Clear field"
+msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:170
+msgid "Search"
+msgstr ""
+
+#: ckanext/ontario_theme/templates/internal/header.html:182
+msgid "close search bar"
```
Comment (Alexandra-Green) [inline ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot:1950]: Really minor nitpick, but this should have a capital c.

Comment (KobikahChandran) [reply ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot:1950]: I swear I did change it, will make that change again with any of your other comments.

Comment (Alexandra-Green) [reply ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot:1950]: Everything else looks good, this is the only thing that requires change

**Review by Alexandra-Green — APPROVED** [review]
Looks good!

**Pull request merged**